doc/userguide/rules/http-keywords.rst

```diff
@@ -787,3 +787,4 @@ Notes
    pattern '<html' is absent from the first inspected chunk.
 
 -  ``file_data`` can also be used with SMTP
```

Comment:
A bit confused wrt why this was left here, but removed from 7 and 8 documentation.

Reply:
It looks like I missed this part when I was fixing up the cherry-pick conflicts (I also apparently forgot the -x). I can get this fixed up.

Reply:
I see, thanks! I'll leave the decision to Victor in this, but IMHO a fixup works.